test/e2e/scalability/load.go

```diff
@@ -289,8 +289,13 @@ var _ = SIGDescribe("Load capacity", func() {
 			framework.Logf("Starting to create %v objects...", itArg.kind)
 			creatingTime := time.Duration(totalPods/throughput) * time.Second
 
+			// Create a waitgroup for profile collection.
+			profileWG := sync.WaitGroup{}
+			profileWG.Add(4)
+
 			createAllResources(configs, creatingTime, testPhaseDurations.StartPhase(200, "load pods creation"))
 			By("============================================================================")
+			go framework.GatherApiserverCPUProfile(&profileWG, "LOAD_creating_RCs")
```

Comment:
Actually, after thinking a bit more about it, I think that running cpu profiler at those times won't be extremely useful in my opinion. Instead, what i would do is to e.g. run a goroutine that would grab the profile e.g. every hour or sth like that. I think that would be more useful.

Reply:
Yeah.. I agree that doing it periodically will make more sense (I changed it now).
Was previously holding back from doing it for couple of reasons:
- we'll have too many profiles (~16 for our 5k-node tests), that too named just by the timestamp
- we'll need to do some 'hardwork' to identify the corresponding profiles from 2 runs before comparing them (sometimes we may not exactly be able to get apples and apples on both sides based on the timing)